python/nav/web/templates/arnold/justifications.html

```diff
@@ -50,8 +50,12 @@
         </div>
 
         {% comment %} Form for creating new justifications {% endcomment %}
-        <div class="medium-6 column">
-            {% crispy form %}
+      <div class="medium-6 column">
+        {% if form.attrs %}
+          {% include 'custom_crispy_templates/flat_form.html' %}
+        {% else %}
+          {% crispy form %}
+        {% endif %}
```

Comment:
The indentations here are different then previously, this should all be indented two more spaces